flashlight/app/objdet/test/PytorchTest.cpp

```diff
@@ -0,0 +1,1231 @@
+#include "flashlight/app/objdet/nn/PositionalEmbeddingSine.h"
+#include "flashlight/pkg/vision/nn/Transformer.h"
+//#include "flashlight/pkg/vision/models/Resnet.h"
+#include "flashlight/pkg/vision/models/Resnet50Backbone.h"
+#include "flashlight/fl/nn/modules/Conv2D.h"
+//#include "flashlight/fl/flashlight.h"
+#include "flashlight/pkg/vision/criterion/Hungarian.h"
+#include "flashlight/pkg/vision/criterion/SetCriterion.h"
+#include "flashlight/pkg/vision/dataset/Coco.h"
+#include "flashlight/pkg/vision/models/Detr.h"
+#include "flashlight/fl/autograd/Variable.h"
+#include "flashlight/fl/autograd/autograd.h"
+#include "flashlight/fl/common/Defines.h"
+#include "flashlight/fl/optim/optim.h"
+
+#include <gtest/gtest.h>
+
+using namespace fl::app::objdet;
+// using namespace fl;
+
+fl::Variable param(af::array x) {
+  return fl::Variable(x, true);
+}
+
+fl::Variable input(af::array x) {
+  return fl::Variable(x, false);
+}
+
+af::array biasReshape(af::array x) {
+  return af::moddims(x, {1, 1, x.dims(0), 1});
+}
+
+bool allClose(
+    const af::array& a,
+    const af::array& b,
+    const double precision = 1e-2) {
+  if ((a.numdims() != b.numdims()) || (a.dims() != b.dims())) {
+    std::cout << " A dims " << a.dims() << std::endl;
+    std::cout << " B dims " << b.dims() << std::endl;
+    std::cout << "Shape mismatch " << std::endl;
+    return false;
+  }
+  std::cout << " Max " << af::max<double>(af::abs(a - b)) << std::endl;
+  return (af::max<double>(af::abs(a - b)) < precision);
+}
+
+using namespace fl::ext::image;
+
+// TEST(Pytorch, basic_conv) {
```

Comment:
I assume this file was already like this, in which case we can ignore, but this might not be something we need to check into the source in general

Reply:
Nope! We should definitely remove.